_docs/reference/config/mixer/policy-and-telemetry-rules.md

```diff
@@ -0,0 +1,190 @@
+metadata:
+  name: rules.config.istio.io
+  labels:
+    package: istio.io.mixer
+    istio: core
+spec:
+  group: config.istio.io
+  names:
+    kind: rule
+    plural: rules
+    singular: rule
+  scope: Namespaced
+  version: v1alpha2
+```
+
+<a name="istio.mixer.v1.config.Action"></a>
+### Action
+Action describes which [Handler](#istio.mixer.v1.config.Handler) to invoke and
+what data to pass to it for processing.
+
+<table>
+ <tr>
+  <th>Field</th>
+  <th>Type</th>
+  <th>Description</th>
+ </tr>
+<a name="istio.mixer.v1.config.Action.handler"></a>
+ <tr>
+  <td><code>handler</code></td>
+  <td>string</td>
+  <td>Required. Fully qualified name of the handler to invoke. Must match the <code>name</code> of a <a href="#istio.mixer.v1.config.Handler">Handler</a>.</td>
+ </tr>
+<a name="istio.mixer.v1.config.Action.instances"></a>
+ <tr>
+  <td><code>instances[]</code></td>
+  <td>repeated string</td>
+  <td>Required. Each value must match the fully qualified name of the <a href="#istio.mixer.v1.config.Instance">Instance</a>s. Referenced instances are evaluated by resolving the attributes/literals for all the fields. The constructed objects are then passed to the <code>handler</code> referenced within this action.</td>
+ </tr>
+</table>
+
+The following example instructs Mixer to invoke the `handler.prometheus` handler
+and pass it the object constructed using the metric instance `RequestCount`.
+
+
+```yaml
+  handler: handler.prometheus
+  instances:
+  - RequestCount.metric
+```
+
+<a name="istio.mixer.v1.config.Handler"></a>
+### Handler
+Handler allows the operator to configure a specific adapter implementation.
+
+In the following example we define a `metric` handler using the Mixer's
+prepackaged [prometheus
+adapter]({{home}}/docs/reference/config/mixer/adapters/prometheus.html). Here,
+we define how the handler should generate prometheus metrics from the metric
+instances provided by Mixer.
+
+
+```yaml
+apiVersion: "config.istio.io/v1alpha2"
+kind: prometheus
+metadata:
+  name: handler
+  namespace: istio-system
+spec:
+  metrics:
+  - name: request_count
+    instance_name: RequestCount.metric.istio-system
+    kind: COUNTER
+    label_names:
+    - source_service
+    - source_version
+    - destination_service
+    - destination_version
+    - response_code
+```
+
+<a name="istio.mixer.v1.config.Instance"></a>
+### Instance
+A Instance tells Mixer how to create values for particular template.
+
+Instance is defined by the operator. Instance is defined relative to a known
+template. Their purpose is to tell Mixer how to use attributes or literals to
+produce values for the specified template at runtime.
+
+The following example instructs Mixer to construct an instance associated with
+[metric template]({{home}}/docs/reference/config/mixer/template/metric.html). It
+provides a mapping from the template's fields to expressions. Instances produced
+with this instance can be referenced by [Actions](#istio.mixer.v1.config.Action)
```

Comment:
with this instance configuration

Reply:
can be referenced by [Actions](#istio.mixer.v1.config.Action) to map handlers to instances

Reply:
updated.